discourse_rock/patches/db_migrations.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/db/post_migrate/20240212034010_drop_deprecated_columns.rb b/db/post_migrate/20240212034010_drop_deprecated_columns.rb
+index 0899da20..586643ba 100644
+--- a/db/post_migrate/20240212034010_drop_deprecated_columns.rb
++++ b/db/post_migrate/20240212034010_drop_deprecated_columns.rb
+@@ -19,6 +19,13 @@ class DropDeprecatedColumns < ActiveRecord::Migration[7.0]
+   }
+
+   def up
++    execute <<~SQL
++    DROP TRIGGER IF EXISTS invites_user_id_readonly ON invites;
```

Comment:
Out of curiosity: in the case of downgrade/roll back, are the triggers recreated?

Reply:
No, because also the columns are not recreated. 
The following is the downgrading function:
```ruby
  def down
    raise ActiveRecord::IrreversibleMigration
  end
```

Reply:
In that case the upgrade document should be updated with a warning about creating a database backup.
Similar to what Wordpress has
https://github.com/canonical/wordpress-k8s-operator/blob/main/docs/how-to/upgrade-wordpress-charm.md